ecl2df/common.py

```diff
@@ -643,26 +643,28 @@ def parse_lyrfile(filename: str) -> List[Dict[str, Any]]:
     #formation-names-description-files-_lyr_
 
     The output has the following format:
```

Comment:
You might only need a double colon at the end here for sphinx to take the following section as verbatim, and then 
you can have the indents as you wish.

Verify the generated doc with `python setup.py build_sphinx` and then `firefox  build/sphinx/html/index.html` from the project root.

Reply:
![image](https://user-images.githubusercontent.com/47847084/118235190-b6cd4f00-b494-11eb-9662-5a6894e462ea.png)

Reply:
I was not able to run `python setup.py build_sphinx`. See error message above